### PR TITLE
changes in Mhwd files

### PR DESCRIPTION
**Pull request by filipe-marques**
changes in Mhwd files, fix tabs for spaces

### CODE REVIEW

**Comment by philmmanjaro** [comment]
seems ok now. @december0123: can you double check this?

**Comment by december0123** [comment]
I apologize for late response.

File: src/Mhwd.hpp [hunk 1]
```diff
@@ -83,8 +83,6 @@ class Mhwd
     std::shared_ptr<Config> getDatabaseConfig(const std::string& configName, const std::string& configType);
     std::shared_ptr<Config> getAvailableConfig(const std::string& configName, const std::string& configType);
 
-    std::vector<Config*> getAllLocalRequirements(Config *config);
```
Comment (december0123) [inline src/Mhwd.hpp]: Dead code. That's good that you removed it.

File: src/Mhwd.cpp [hunk 2]
```diff
@@ -614,7 +619,7 @@ bool Mhwd::runScript(std::shared_ptr<Config> config, MHWD::TRANSACTIONTYPE opera
     for (auto&& foundDevice = foundDevices.begin();
             foundDevice != foundDevices.end(); ++foundDevice)
     {
-        bool found = false;
+        static bool found = false;
```
Comment (december0123) [inline src/Mhwd.cpp]: That change just makes the reader confused. You stumble across it and begin to wonder why did someone make that variable static. We don't save memory (found is a bool which is a primitive type - no dynamic memory allocation), we just make the code more confusing and I don't think that's the way to go.

File: src/Mhwd.cpp [hunk 3]
```diff
@@ -564,21 +565,25 @@ MHWD::STATUS Mhwd::uninstallConfig(Config *config)
     }
     else
     {
-        // TODO: Should we check for local requirements here?
-
-        // Run script
-        if (!runScript(installedConfig, MHWD::TRANSACTIONTYPE::REMOVE))
+        // Should we check for local requirements here?
+        if(!localRequirements.empty())
```
Comment (december0123) [inline src/Mhwd.cpp]: Local requirements are checked before this function is called. We don't need that check here.

File: src/Mhwd.cpp [hunk 4]
```diff
@@ -552,6 +552,7 @@ MHWD::STATUS Mhwd::installConfig(std::shared_ptr<Config> config)
 MHWD::STATUS Mhwd::uninstallConfig(Config *config)
 {
     std::shared_ptr<Config> installedConfig{getInstalledConfig(config->name_, config->type_)};
+    std::vector<std::shared_ptr<Config>> localRequirements{data_.getAllLocalRequirements(installedConfig)};
```
Comment (december0123) [inline src/Mhwd.cpp]: Local requirements are checked before this function is called. We don't need that check here.

File: src/Mhwd.cpp [hunk 5]
```diff
@@ -564,21 +565,25 @@ MHWD::STATUS Mhwd::uninstallConfig(Config *config)
     }
     else
     {
-        // TODO: Should we check for local requirements here?
-
-        // Run script
-        if (!runScript(installedConfig, MHWD::TRANSACTIONTYPE::REMOVE))
+        // Should we check for local requirements here?
```
Comment (december0123) [inline src/Mhwd.cpp]: That comment should be removed long time ago. You can see that it only confused you (that's why comments should be the last resort and should not be widely used).

File: README.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
+## Manjaro HardWare Detection (mhwd)
```
Comment (december0123) [inline README.md]: I think it's fine for now. It's a good idea to finally have a proper readme :+1:

**Comment by philmmanjaro** [comment]
I think with these out you can now proper your commit as suggested by @december0123 and he or I will merge your changes ...

**Comment by december0123** [comment]
It's ok now. Thanks.

**Pull request merged**